modules/minimint-mint/src/lib.rs

```diff
@@ -612,13 +616,6 @@ impl Mint {
             partial_sig,
         );
 
-        // FIXME: add own id to cfg
-        if peer == self.key_id {
```

Comment:
This subtly changes behavior: from what I gather we accept that we will be resumbitting our share till a signature has been produced (even if our share was already in a consensus outcome). The maximum amount of rounds this will happen is `n-t` after your "banning bad peers" PR I think, so maybe ok.

Reply:
Yeah I figured resubmitting during a banning epoch isn't a problem.

Reply:
Yeah, it should basically never happen.